COPY/etc/issue.template

```diff
@@ -1,4 +1,4 @@
-Welcome to the \P Virtual Appliance.
+Welcome to the Virtual Appliance.
```

Comment:
all other linux systems don't have the name of their product at the linux prompt...